inputs/templates/test/MakeCohortVcf/MainVcfQc.json.tmpl

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
For consistency can you move this to `inputs/templates/test/MainVcfQc/`? Easier to find that way I think